py/selenium/webdriver/common/driver_finder.py

```diff
@@ -40,7 +40,7 @@ def get_path(service: Service, options: BaseOptions) -> str:
             msg = f"Unable to obtain driver for {options.capabilities['browserName']} using Selenium Manager."
             raise NoSuchDriverException(msg) from err
 
-        if path is None or not Path(path).is_file():
```

Comment:
Why is this needed?

Reply:
It should not be needed, I'm moving the path expansion into the driver-specific code.